### PR TITLE
#165305716 Setup Test Coverage on Codeclimate

### PR DESCRIPTION
**Pull request by matthenge**
### What does this PR do?
Setup Code climate Test Coverage.

### What are the tasks to be completed?
- Add Test coverage badge
- Add Maintainability badge

### How can this be manually tested?
After cloning the repository:
```
$ git checkout ch-setup-test-coverage-165305716
$ npm run start:dev
```

### What are the relevant pivotal tracker stories?
[#165305716](https://www.pivotaltracker.com/story/show/165305716)

### Screenshot
image

### CODE REVIEW

**Comment by Philipotieno** [comment]
This looks okay to me
You can squash the commits before the PR is closed

**Review by Philipotieno — APPROVED** [review]

**Review by Ogutu-Brian — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "test": "jest --coverage",
     "build": "webpack -p ",
     "start:dev": "webpack -d && webpack-dev-server --mode development --open --hot",
-    "lint": "./node_modules/.bin/eslint"
+    "lint": "./node_modules/.bin/eslint",
+    "coverage": "jest --coverage"
```
Comment (koechkevin) [inline package.json]: @matthenge This command will not be necessary anymore. It is similar to the command in line 7

**Review by koechkevin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (matthenge) [reply package.json]: Okay @koechkevin let me make the necessary changes

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]
Hello @matthenge ,
Would you exclude tests files from being checked by code climate? Most times, test files have similar blocks of code that would be recognised by code climate as duplicates.
To achieve this, you can use [this resource](https://docs.codeclimate.com/docs/excluding-files-and-folders)

File: package.json [hunk 2]
```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "@babel/core": "^7.4.5",
     "@babel/polyfill": "^7.4.4",
+    "eslint-plugin-react": "^7.13.0",
```
Comment (Ogutu-Brian) [inline package.json]: Hello @matthenge ,
This package is already in devDependencies (line 32 of package.json). You can remove it to avoid redanduncy

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (matthenge) [reply package.json]: @Ogutu-Brian Done

**Review by koechkevin — CHANGES_REQUESTED** [review]
@matthenge your branch is not up to date

**Review by Amoswachira — APPROVED** [review]

**Review by Ogutu-Brian — APPROVED** [review]

**Review by koechkevin — APPROVED** [review]

**Pull request merged**